### PR TITLE
amd64/main.c: add a post function to improve bringup debugging

### PR DESCRIPTION
**Pull request by rminnich**
harvey does not boot with ease on many modern x86 CPUs yet.
Add a post() function for main to provide copious debug
output, if a static variable is set.

Signed-off-by: Ronald G Minnich <rminnich@gmail.com>

### CODE REVIEW

File: sys/src/9/amd64/main.c [hunk 1]
```diff
@@ -541,9 +566,9 @@ main(uint32_t mbmagic, uint32_t mbaddress)
 	active.exiting = 0;
 	active.nbooting = 0;
 
-	asminit();
-	multiboot(mbmagic, mbaddress, 0);
-	options(oargc, oargv);
+	asminit(); post("	asminit();", postterminal++);
+	multiboot(mbmagic, mbaddress, 0); post("	multiboot(mbmagic, mbaddress, 0);", postterminal++);
```
Comment (gmacd) [inline sys/src/9/amd64/main.c:570]: What's up with the tabbing here? Intentional?

**Review by gmacd — APPROVED** [review]

File: sys/src/9/amd64/main.c (hunk 1 above)
Comment (rminnich) [reply sys/src/9/amd64/main.c:570]: yeah, it's so via grep I can look for only those lines.

**Comment by rminnich** [comment]
I've used this just now on an AMD Zen CRB and ... it is a timesaver to say the least.

**Comment by orangecms** [comment]
Nice! I need to try it on the A300 :)

**Pull request merged**